tests/src/Kernel/fixtures/markup_rendering_patterns/inpage_navigation.yml

```diff
@@ -75,3 +75,22 @@ inpage_navigation_non_full:
       'a[href="#item-3"]': 'Heading 3'
       'a[href="#item-4"]': 'Heading 4'
       'h5': 'Page content'
+inpage_navigation_with_no_links:
```

Comment:
the there is not '#attached' or 'script' available to assert against, so this will have to do